package.json

```
@@ -99,7 +99,7 @@
     "eslint": ">= 6"
   },
   "engines": {
-    "node": "10.* || 12.* || >= 14"
+    "node": "12.* || >= 14"
```

Here's how we can drop Node 15 support (which is end-of-life already).

```suggestion
    "node": "12.* || 14.* || >= 16"
```

https://nodejs.org/en/about/releases/